packages/clerk-js/package.json

```diff
@@ -94,5 +94,6 @@
   },
   "publishConfig": {
     "access": "public"
-  }
+  },
+  "legacybrowserslist": "Chrome > 73, Firefox > 66, Safari > 12, iOS > 12, Edge > 18, Opera > 58"
```

Comment:
I was hoping we could keep this next to the `"browserslist"` property, but the plugin we use to sort fields in `package.json` files can't be configured that way without specifying the order of _all_ fields.

Reply:
How about `"browserslistLegacy"` then?